MinecraftClient/Commands/Move.cs

```diff
@@ -41,26 +46,28 @@ public override string Run(McClient handler, string command, Dictionary<string,
                         case "north": direction = Direction.North; break;
                         case "south": direction = Direction.South; break;
                         case "get": return handler.GetCurrentLocation().ToString();
-                        default: return Translations.Get("cmd.look.unknown", argStr);
+                        default: return Translations.Get("cmd.look.unknown", args[0]);
                     }
                     if (Movement.CanMove(handler.GetWorld(), handler.GetCurrentLocation(), direction))
                     {
-                        handler.MoveTo(Movement.Move(handler.GetCurrentLocation(), direction));
-                        return Translations.Get("cmd.move.moving", argStr);
+                        if (handler.MoveTo(Movement.Move(handler.GetCurrentLocation(), direction), allowUnsafe: takeRisk))
+                            return Translations.Get("cmd.move.moving", args[0]);
+                        else return !takeRisk ? Translations.Get("cmd.move.suggestforce") : Translations.Get("cmd.move.dir_fail");
                     }
                     else return Translations.Get("cmd.move.dir_fail");
                 }
-                else if (args.Length == 3)
+                else if (args.Count == 3)
                 {
                     try
                     {
                         int x = int.Parse(args[0]);
                         int y = int.Parse(args[1]);
                         int z = int.Parse(args[2]);
                         Location goal = new Location(x, y, z);
-                        if (handler.MoveTo(goal))
+
+                        if (handler.MoveTo(goal, allowUnsafe: takeRisk))
                             return Translations.Get("cmd.move.walk", goal);
-                        return Translations.Get("cmd.move.fail", goal);
+                        else return !takeRisk ? Translations.Get("cmd.move.suggestforce") : Translations.Get("cmd.move.fail", goal);
```

Comment:
How about removing the negation in ternary condition and also include coordinates:
```suggestion
                        else return takeRisk ? Translations.Get("cmd.move.fail", goal) : Translations.Get("cmd.move.suggestforce", goal);
```
(see below for coordinates in message)